pkgs/servers/xmpp/ejabberd/default.nix

```diff
@@ -27,7 +27,7 @@ in stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://www.process-one.net/downloads/downloads-action.php?file=/${version}/ejabberd-${version}.tar.gz";
-    sha256 = "sha256-K4P+A2u/Hbina4b3GP8T3wmPoQxiv88GuB4KZOb2+cA=";
+    sha256 = "b/HUGh/2JhoMhGGTZH2OwUPoIUKFm/HP3GIpkCLOsq0=";
```

Comment:
Please use the new hash format, `hash = "sha256-xyz...";`

Reply:
done